packages/airnode-starter/README.md

```diff
@@ -0,0 +1,193 @@
+
+## Setup
+
+Follow the [repository instructions](https://github.com/api3dao/airnode#instructions).
+Also, make sure you have `yarn` installed.
+If you want to run Airnode in as docker container, you'll also need to have `docker` installed.
+
+## Instructions
+
+The following instructions will guide you step by step with RRP flow. Please note, that some steps may be skipped
+depending on how you want to run the example - e.g. you don't need to start hardhat network locally when you intend to
+run the example on Rinkeby.
+
+### 1. Choose an example
+
+First, you need to choose what example you want to run. We call these examples "integrations" because they are
+integrated to some API. You will also need to choose how you want to run the Airnode and which network to use. If you intend to run an integration on Rinkeby, you will also need a funded account to deploy necessary contracts and trigger an Airnode request.
+
+Run the following script and provide this information:
+```sh
+yarn choose-integration
+```
+
+### 2. (Only if using local blockchain) Start hardhat network
+
+Run:
+```sh
+yarn eth-node
+```
+
+which will spin up a [hardhat network](https://hardhat.org/hardhat-network/) on your machine.
+
+### 3. Deploy the RRP contract
+
+The RRP is a contract that through with the [requester](link) triggers a request for Airnode. This contract is common for all Airnodes and requesters on a chain.
+
+API3 team will deploy this contract for the most popular chains at some point, but for now you have to deploy it yourself using:
+
+```sh
+yarn deploy-rrp
+```
+
+### 4. (Only if deploying to AWS) Create AWS secrets file
+
+If you intend to deploy Airnode on AWS, you will need to specify the credentials which will be used by the [airnode
+deployer](link). If you are not sure where to find these or how to create an AWS account, follow [this
+video](https://www.youtube.com/watch?v=KngM5bfpttA).
+
+After you know the secrets, run the following script to specify them.
+
+```sh
+yarn create-aws-secrets
+```
+
+The command will generate an `aws.env` file with the entered secrets. This file be used only by the deployer and will
+never leave your machine.
+
+### 5. Create Airnode secrets
+
+Airnode is configured by two files - `config.json` and `secrets.env`. Run the script below to create the `secrets.env` file with the necessary values.
+
+```sh
+yarn create-airnode-secrets
+```
+
+_Refer to the [documentation](https://docs.api3.org/airnode/next/grp-providers/guides/build-an-airnode/configuring-airnode.html) for more details_
+
+### 6. Build docker artifacts
+
+Our docker images are based on a common container which we call "artifacts". This intermediate container is then used by
+both [deployer](https://github.com/api3dao/airnode/tree/master/packages/deployer) and
+[airnode](https://github.com/api3dao/airnode/tree/master/packages/node). You can build the artifacts container by running:
+
+```sh
+yarn rebuild-artifacts-container
```

Comment:
Encountered `#11 269.0 @api3/protocol: Error HH502: Couldn't download compiler versions list. Please check your connection.` initially. Reran the command and it was fine 🤷

Reply:
Seems to me like an network error? o.O